lib/core/test/unit/Cardano/Wallet/DBSpec.hs

```diff
@@ -639,10 +652,16 @@ prop_parallelPut putOp readOp resolve dbLayer (KeyValPairs pairs) =
         res <- once pairs (readOp db . fst)
         length res `shouldBe` resolve pairs
 
+data SkipTests
+    = RunAllTests
+    | SkipTxHistoryReplaceTest
+    deriving (Show, Eq)
```

Comment:
This looks a bit dodgy :no_mouth: . What exactly is the issue with this test? If there's an issue, let's disable it temporarily and open a bug ticket about fixing it instead?